caesars-palace/blocks/intro-section/intro-section.js

```diff
@@ -0,0 +1,21 @@
+import { createOptimizedPicture } from '../../scripts/lib-franklin.js';
+
+export default function decorate(block) {
+  const rows = [...block.children];
```

Comment:
You may want to consider using `querySelector` instead.

Something like this should target the first child div of the block.
```
const content = block.querySelector(':scope > div:first-of-type');
```

Reply:
I based this portion of the script off of `cards.js` and a few others where they're doing something similar, since I will need to iterate over the rows (i.e. will need to get the second row).

I've noticed that none of the blocks seem to contain much error handling.  Is that something we're not concerned about?  Are we assuming it'll get caught by the authors during preview?  There's not much of a mechanism to provide feedback to the author and I don't think we want live blocks to provide authoring error messages.

Reply:
The difference with cards is that any number of card items are supported which requires iterating over each item. In an intro section however there are a specific set of rows that must be defined so you don't really need to iterate over an unknown set of rows. Using `first-of-type` and `last-of-type` would allow you to select the two rows you care about with out having to convert all the children to an array. In the end both methods work however so keeping what you have is fine.

Reply:
Makes sense.  Thank you for the explanation. :)